projects/igniteui-angular/src/lib/core/styles/components/column-actions/_column-actions-theme.scss

```diff
@@ -51,6 +51,7 @@
     @return extend($theme, (
         name: $name,
         palette: $palette,
+        elevation: $elevations,
```

Comment:
This should be named `elevations` (plural), instead of the singular `elevation`.